client/src/modules/vouchers/complex-voucher.ctrl.js

```diff
@@ -323,5 +323,11 @@ function ComplexJournalVoucherController(
       .catch(Notify.handleError);
   }
 
+  // help to group transaction_types
+  vm.groupByType = (item) => {
+    const type = `FORM.LABELS.${item.type.toUpperCase()}`;
```

Comment:
A shared definition for this mapping can be found in `bhConstans` here: 
https://github.com/IMA-WorldHealth/bhima-2.X/blob/c94a1b4dbd1dac5d476b364c60f3d361dba906ae/client/src/js/constants/bhConstants.js#L150

Using this would mean that if the JSON translation structure or the database changes we only have to update the application in one place. 

It can be used by requiring the constants library: 
```js
Controller.$inject = ['bhConstants'];

function Controller(bhConstants) { 
  const type = bhConstants.transactionTypeMap[item.type];
}
```